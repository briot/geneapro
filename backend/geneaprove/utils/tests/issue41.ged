0 HEAD
1 SOUR test for github issue #38
1 CHAR UTF-8
1 GEDC
2 VERS 5.5.1
2 FORM LINEAGE-LINKED
1 SUBM @SUBM@
0 @SUBM@ SUBM
1 NAME Individual Tester
1 LANG English
0 @I1@ INDI
1 NAME Individual Middle /Tester1/
1 RESI abt 1950
0 TRLR
