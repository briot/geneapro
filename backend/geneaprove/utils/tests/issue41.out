<dir>/issue41.ged:13 Unexpected text value after RESI, expected 'Y'
OK
